src/rtc/rtc_s1_single_job.py

```diff
@@ -13,6 +13,7 @@
 import argparse
 
 import isce3
+from scipy import ndimage
```

Comment:
Could not leave the comment for the line #5, but it seems the imported module `datetime` was not used in this code. Can you check?